### PR TITLE
[Next.js][React]  Link component does not add anchor to the internal links

### PR DESCRIPTION
**Pull request by sc-addypathania**
Add support for anchor attribute for:

- sitecore-jss-react
- sitecore-jss-nextjs

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by illiakovalenko** [comment]
@sc-addypathania Please, double check failed build

**Comment by illiakovalenko** [comment]
@sc-addypathania Please, update CHANGELOG

**Review by illiakovalenko — CHANGES_REQUESTED** [review]
Please, update CHANGELOG, otherwise looks good
And I guess that you don't need to cancel azure pipelines build when you are in the PR, you need to that when you merged your changes to release branch

**Comment by sc-addypathania** [comment]
> Please, update CHANGELOG, otherwise looks good And I guess that you don't need to cancel azure pipelines build when you are in the PR, you need to that when you merged your changes to release branch

I will do that.
I didn't cancel the pipelines. One of the test is failing. I am looking into that.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,21 +1,32 @@
...
+`[sitecore-jss-react]` `[sitecore-jss-nextjs]`
+
+- [Next.js][react] Link component does not add anchor to the internal links ([#1228](https://github.com/Sitecore/jss/pull/1228))
+
 ## 19.0.2
 
 ## Bug Fixes
 
 `[sitecore-jss-vue]`
-* [EE] Cannot add a new rendering to the newly created ([#902](https://github.com/Sitecore/jss/pull/902))
```
Comment (ambrauer) [inline CHANGELOG.md:11]: Looks like your editor made some other formatting changes automatically. Can you please revert these? Maybe stick with the "*" bullet (what we've used thus far) for the new content too.

**Review by ambrauer — APPROVED** [review]
See comment about CHANGELOG, otherwise looks good 👍

**Pull request merged**